lbjava/src/main/java/edu/illinois/cs/cogcomp/lbjava/learn/SupportVectorMachine.java

```diff
@@ -72,6 +77,11 @@
     public static final double defaultEpsilon = 0.1;
     /** Default for {@link #bias}. */
     public static final double defaultBias = 1.0;
+    /** any weight less than this is considered irrelevant. This is for prunning. */
+    public static final double defaultFeaturePruningThreshold = 0.000001;
```

Comment:
Is this always used in an absolute sense? I.e. it always have to be positive. 
In other words, if its zero, no pruning will be done.

Reply:
@danyaljj setting the threshold to zero is how you disable pruning, effectively.